### DISCUSSION TITLE
Shortcut to copy/paste commands in terminal?

### DISCUSSION THREAD

**Discussion by abitrolly** [ohmyzsh/ohmyzsh#11077] [Help] [answered]
Right now I am using trackpoint to select and context menu to copy commands for saving them into a script. Maybe there is shortcut for that? I couldn't find the reference.

**Comment by carlosala** [comment]
If you want to copy the latest command that you executed, for example, you can use:
```sh
echo "!!" | clipcopy
```
Note that this is only going to work if you are using `ohmyzsh`, as `clipcopy` is defined there 👌🏻 

You can also go to `~/.zsh_history` file to copy other commands that you executed previously.

**Reply by abitrolly** [reply]
This kind of works, but it is still not as convenient as keyboard shortcut.
```
echo !! | wl-copy
```

**Comment by abitrolly** [comment, marked as answer]
I actually found that [`copybuffer`](https://github.com/ohmyzsh/ohmyzsh/edit/master/plugins/copybuffer/README.md) does what I need.

> binds the ctrl-o keyboard shortcut to a command that copies the text that is currently typed in the command line
